googler

```diff
@@ -334,6 +334,15 @@ except GetoptError as e:
     print("googler:", e)
     sys.exit(1)
 
+# Set up logging
```

Comment:
> as I said, if debug is basically implicit in logger.debug.
Understood. I meant the initialization too (and without any extra complexity).
> Feel free to close this PR.
Yep, I think we can skip this, at least for now.